src/test/kotlin/notification/mothers/NotificationMother.kt

```diff
@@ -6,6 +6,26 @@ import shared.mothers.IdentifierMother
 
 object NotificationMother {
 
-    fun getValidNotification() = Notification(id = IdentifierMother.getValidIdentifier())
+    fun getValidNotification() = Notification(
```

Comment:
This method doesn't have any sense because why we would want to create a valid notification without noticing which note has this notification?

The sense would be to
```kotlin
    fun getValidNotificationFor(note: Note) = Notification( noteId = NoteMother.getIdentifierFrom(note), .....)
```

Please refer to discussion #43 for this.